src/shared/constants.ts

```diff
@@ -16,7 +16,7 @@ export const samAboutInstallUrl: string = 'https://aws.amazon.com/serverless/sam
 export const vscodeMarketplaceUrl: string =
     'https://marketplace.visualstudio.com/items?itemName=AmazonWebServices.aws-toolkit-vscode'
 export const githubUrl: string = 'https://github.com/aws/aws-toolkit-vscode'
-export const reportIssueUrl = `${githubUrl}/issues/new/choose`
+export const createIssueOnGitHubUrl = `${githubUrl}/issues/new/choose`
```

Comment:
this is consistent with `githubUrl` just above, and is less awkward:

```suggestion
export const githubCreateIssueUrl = `${githubUrl}/issues/new/choose`
```

Reply:
Done.